libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -148,6 +150,8 @@ static RCInput_DSM rcinDriver;
 static RCInput_SBUS rcinDriver;
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_NAVIO2
 static RCInput_Navio2 rcinDriver;
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_AERO
```

Comment:
combine with HAL_BOARD_SUBTYPE_LINUX_DISCO just like the NAVIO, ERLE, BH, PXFMINI one does?